Parsers/ASimAuthentication/ProductParsers/ASimAuthenticationPostgreSQL.yaml

```diff
@@ -27,14 +27,17 @@ ParserQuery: |
     EventVendor = 'PostgreSQL'
     , EventProduct = 'PostgreSQL'
     , EventCount = int(1)
+    , EventSchema = 'Authentication'
     , EventSchemaVersion = '0.1.1'
     , EventResult = 'Success'
     , EventStartTime = TimeGenerated
     , EventEndTime = TimeGenerated
-    , EventType = 'Authorization'
+    , EventType = 'Logon'
+    , DvcHostname = Computer
+    , DvcIpAddr = extract(@'\d{1,3}\.\d{1.3}\.\d{1,3}\.\d{1,3}', 1, Computer)
     , TargetUsernameType = 'Simple'
     , TargetUsername = extract(@'user=(.*?)\sdatabase', 1, RawData)
-    , EventMessage = 'Connection authorized'
+    , EventResultDetails = 'Connection authorized'
```

Comment:
EventResultDetails has to be mapped to one of the [allowed values](https://docs.microsoft.com/azure/sentinel/authentication-normalization-schema#common-fields-with-specific-guidelines)